client/layout/guided-tours/config-elements.js

```diff
@@ -76,7 +79,13 @@ export class Tour extends Component {
 export class Step extends Component {
 	static propTypes = {
 		name: PropTypes.string.isRequired,
-	}
+		placement: PropTypes.string,
+		next: PropTypes.string,
+		target: PropTypes.string,
+		arrow: PropTypes.string,
```

Comment:
OTOH `.oneOf( [ 'below', 'beside', ... ] )` would probably be sufficient in both cases.